projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -118,30 +118,35 @@ def download_job(name: str, team: str, path: str):
         return f"Downloaded the job with name {name} to {path}. "
 
     @staticmethod
-    def create_job(name: str, team: str, path: str, local: bool, cloud: bool):
+    def create_job(name: str, team: str, path: str):
         """
         Execute `create job`.
         :param name: the name of the data job that will be created
         :param team: the team of the data job that will be created
         :param path: the path to the directory where the job will be created
-        :param local: create sample job on local file system
-        :param cloud: create job in the cloud
         :return: message that the job is created
         """
-        cmd = JobCreate(RestApiUrlConfiguration.get_rest_api_url())
-        if cloud:
-            cli_utils.check_rest_api_url(RestApiUrlConfiguration.get_rest_api_url())
-
-        if local:
-            cmd.validate_job_path(path, name)
-
         jupyter_job_dir = os.path.abspath(
             os.path.join(os.path.dirname(__file__), "jupyter_sample_job")
         )
-
-        cmd.create_job(name, team, path, cloud, local, jupyter_job_dir)
-
-        return f"Job with name {name} was created."
+        try:
+            cmd = JobCreate(RestApiUrlConfiguration.get_rest_api_url())
+            cli_utils.check_rest_api_url(RestApiUrlConfiguration.get_rest_api_url())
+            cmd.validate_job_path(path, name)
```

Comment:
That's redundant , JobCreate already validates job path

Reply:
fixed